### ISSUE TITLE
6-1 Softmax classification: Multinomial classification

### ISSUE THREAD

**Issue by NahahaNahaha** [NahahaNahaha/tensorflow#12]
일단 지금까지 배운 것에서 
H(x)=WX 라 하면 이것의 출력값을 0과 1 사이로 조정하기 위하여 g(Z)라는 함수를 도입했다. 
g(z)= 1/(1+e^(-2)) 같은 형태 
Logistic regression 이란 데이터들의 집합을 가르는 직선을 의미한다고 볼 수 있는데 데이터들의 처리값을 2개 중에 하나로 표현하고 싶을 때 이용하는 것이었다. ex) 0 and 1 
허나 A,B,C와 같이 학점이 결과값일 때는 한 개의 직선으로는 표현할 수 없기 때문에 3개의 직선을 긋는다. 
이것이  Multinomial classification이다 
<img width="637" alt="2018-04-04 1 01 02" src="https://user-images.githubusercontent.com/26895961/38260882-b12cf0e4-37a3-11e8-959b-d9847b1ed9fe.png">
<img width="1064" alt="2018-04-04 1 01 28" src="https://user-images.githubusercontent.com/26895961/38260923-ce68a1e4-37a3-11e8-9cd8-7c1deccce483.png">
다음과 같이 표현할 수 있다. (위 예시에서 x,y,z들의 weight가 구분되어 있지 않는건 교수님의 실수같다)
<img width="476" alt="2018-04-04 1 01 37" src="https://user-images.githubusercontent.com/26895961/38260931-d0378044-37a3-11e8-9876-00d1e0aa0a24.png"> 
식이 matrix의 형태로 표현되어있으므로 이렇게 합쳐서 나타낼 수 있다. 

하지만 이렇게 나타낼 경우 위에 있는 sigmod 의 처리를 할 수 없다.(3개가 하나가 되었으므로) 
이에 대해서는 다음 시간에 배운다고 한다. 생각해보면 sigmod의 종합치를 구하지 않을까 싶다
